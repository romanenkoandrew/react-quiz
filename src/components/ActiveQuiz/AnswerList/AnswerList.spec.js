import React from "react";
import { shallow } from "enzyme";
import AnswerList from "./AnswerList";

describe("AnswerList component", () => {
  it("should render AnswerList with props.state", () => {
    const component = shallow(
      <AnswerList
        state={true}
        answers={[{ answer: { text: "test", id: "test" } }]}
      />
    );
    expect(component).toMatchSnapshot();
  });
  it("should render AnswerList without props.state", () => {
    const component = shallow(
      <AnswerList
        state={false}
        answers={[{ answer: { text: "test", id: "test" } }]}
      />
    );
    expect(component).toMatchSnapshot();
  });
});
