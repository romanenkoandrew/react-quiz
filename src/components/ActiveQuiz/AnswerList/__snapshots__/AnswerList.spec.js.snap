// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnswerList component should render AnswerList with props.state 1`] = `
<ul
  className="AnswerList"
>
  <AnswerItem
    answer={
      Object {
        "answer": Object {
          "id": "test",
          "text": "test",
        },
      }
    }
    key="0"
  />
</ul>
`;

exports[`AnswerList component should render AnswerList without props.state 1`] = `
<ul
  className="AnswerList"
>
  <AnswerItem
    answer={
      Object {
        "answer": Object {
          "id": "test",
          "text": "test",
        },
      }
    }
    key="0"
    state={null}
  />
</ul>
`;
